packages/shared/src/types/abTests/banner.ts

```diff
@@ -38,16 +38,14 @@ export interface BannerVariant extends Variant {
     separateArticleCount?: boolean;
 }
 
-export type CanRun = (targeting: BannerTargeting, pageTracking: PageTracking) => boolean;
-
 export type BannerTestGenerator = () => Promise<BannerTest[]>;
 
 export interface BannerTest extends Test<BannerVariant> {
     name: string;
+    status: TestStatus;
     bannerChannel: BannerChannel;
     isHardcoded: boolean;
     userCohort: UserCohort;
-    canRun: CanRun;
```

Comment:
`canRun` isn't being used currently so I've removed it. We can always bring it back later if necessary

Reply:
I think it's quite nice to remove it, now the `BannerTest` object is just data